jrnl/plugins/meta.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import importlib
+import pkgutil
+
+import jrnl.contrib.exporter
+import jrnl.contrib.importer
+import jrnl.plugins.exporter
+import jrnl.plugins.importer
+
+from .template_exporter import __all__ as template_exporters
+
+__exporters_builtin = list(
+    pkgutil.iter_modules(
+        jrnl.plugins.exporter.__path__, jrnl.plugins.exporter.__name__ + "."
+    )
+)
+__exporters_contrib = list(
+    pkgutil.iter_modules(
+        jrnl.contrib.exporter.__path__, jrnl.contrib.exporter.__name__ + "."
+    )
+)
+
+__importers_builtin = list(
+    pkgutil.iter_modules(
+        jrnl.plugins.importer.__path__, jrnl.plugins.importer.__name__ + "."
+    )
+)
+__importers_contrib = list(
+    pkgutil.iter_modules(
+        jrnl.contrib.importer.__path__, jrnl.contrib.importer.__name__ + "."
+    )
+)
```

Comment:
Instead of duplicating each of these (one for builtin, and one for contrib), can we have a function that will do all of this work given a path? Then we could run that function once for each path (builtin, and contrib).

Reply:
A standalone function would also be easier to add tests for.

Reply:
Can be done!